src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -88,8 +88,6 @@ jobs:
         variables: |
             Flip: Bling
             Fizz: Buzz
-        git_ref: ${{ github.ref }}
-        git_commit: ${{ github.sha }}
```

Comment:
Turns out these values are not actually required for this action! They don't break it but they're just being ignored and causing a warning message.